README

```diff
@@ -485,7 +485,6 @@ dosbox -opencaptures program
         - pcjr          - IBM PCjr
         - tandy         - Tandy Graphics Adapter (Tandy 1000)
         - ega           - IBM Enhanced Graphics Adapter
-        - vgaonly       - IBM Video Graphics Array (see below)
```

Comment:
My suggestion: Keep `svga_et4000` as it is and integrate `vgaonly` only to `svga_paradise`. Then once we get `vgaonly` into `svga_paradise`, we should make `machine = vgaonly` redirect the user to (or have the same effect off) `machine = svga_paradise`.

Reply:
Suggestion based on the information provided by @joncampbell123 and @Grounded0

Reply:
(Follow the conversation above to understand why I'm suggesting this)

Reply:
Yeah, just read Jon's comment again about chained modes, and `svga_paradise` is perhaps a better candidate, agreed.

Reply:
`svga_paradise` seems good for now,